src/commands/poll.js

```diff
@@ -103,17 +145,78 @@ exports.run = (client, message, args, callback) => {
     }
 
     let channel = options.channel ? client.guilds.cache.get(config.ids.guild_id).channels.cache.get(config.ids.votes_channel_id) : message.channel;
+    if(options.channel && options.delayed) {
+        return callback("Du kannst keine verzögerte Abstimmung außerhalb des Umfragenchannels machen!");
+    }
 
     /** @type {import("discord.js").TextChannel} */
     (channel).send(/** @type {Object} embed */(embed))
         .then(async msg => {
             message.delete();
             for (let i in pollOptions) await msg.react(EMOJI[i]);
+
+            if(options.delayed) {
+                const reactionMap = [];
+                /** @type {string[][]} */
+                const reactions = [];
+                pollOptions.forEach((option, index) => {
+                    reactionMap[index] = option;
+                    reactions[index] = [];
+                });
+
+                exports.delayedPolls.push({
+                    pollId: msg.id,
+                    createdAt: new Date(),
+                    finishesAt: finishTime,
+                    reactions,
+                    reactionMap
+                });
+            }
         });
 
     return callback();
 };
 
+/**
+ * Initialized crons for delayed polls
+ * @param {import("discord.js").Client} client
+ */
+exports.startCron = (client) => {
+    cron.schedule("* * * * *", async() => {
+        const currentDate = new Date();
+        const pollsToFinish = exports.delayedPolls.filter(delayedPoll => currentDate >= delayedPoll.finishesAt);
+        /** @type {import("discord.js").TextChannel} */
+        const channel = client.guilds.cache.get(config.ids.guild_id).channels.cache.get(config.ids.votes_channel_id);
+
+        for(let i = 0; i < pollsToFinish.length; i++) {
```

Comment:
why no `.forEach` ? :eyes:

Reply:
Because I don't want to run n asynchronous tasks when using discord API calls.
It's better to handle one after another.